tensorboard/webapp/widgets/data_table/data_table_component.scss

```diff
@@ -12,16 +12,21 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
+@use '@angular/material' as mat;
+@import 'tensorboard/webapp/theme/tb_theme';
 
 .data-table {
   border-spacing: 4px;
   font-size: 13px;
 
   th {
-    background-color: white;
+    background-color: mat.get-color-from-palette($tb-background, background);
     position: sticky;
     text-align: left;
     top: 0;
+    @include tb-dark-theme {
```

Comment:
nit: could you add a new line between 26-27?
I think it's cleaner to separate them.
https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/plugins/plugins_component.scss;l=23?q=%22@include%20tb-dark-theme%20%7B%22&ss=tensorflow%2Ftensorboard